resolwe_bio/processes/samtools/samtools_coverage.py

```diff
@@ -0,0 +1,162 @@
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class SamtoolsCoverage(Process):
+    """Samtools coverage.
+
+    Computes the depth at each position or region and creates tabulated text.
+
+    For more information about samtools coverage, click
+    [here](https://www.htslib.org/doc/samtools-coverage.html).
+    """
+
+    slug = "samtools-coverage"
+    process_type = "data:samtoolscoverage"
+    name = "Samtools coverage"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/rnaseq:6.0.0"},
+        },
+        "resources": {
+            "cores": 2,
+            "memory": 16384,
+        },
+    }
+    category = "Samtools"
+    data_name = "Samtools coverage"
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+
+    class Input:
+        """Input fields for SamtoolsView."""
+
+        bam = ListField(
+            DataField(data_type="alignment:bam"),
+            label="Input BAM files",
+            description="Select BAM file(s) for the analysis. Coverage information will "
+            "be calculated from the merged alignments.",
+        )
+        region = StringField(
+            label="Region",
+            description="Regions can be specified as: RNAME:STARTPOS-ENDPOS "
+            "and all position coordinates are 1-based, where RNAME is the "
+            "name of the contig. If the input BAM file was generated by "
+            "General RNA-seq pipeline, you should use only chromosome "
+            "numbers to subset the input file, e.g. 3:30293-39103.",
+            required=False,
+        )
+
+        class AdvancedOptions:
+            """Advanced options."""
+
+            min_read_length = IntegerField(
+                label="Minimum read length",
+                required=False,
+                description="Ignore reads shorter than specified number of "
+                "base pairs",
+            )
+            min_mq = IntegerField(
+                label="Minimum mapping quality",
+                required=False,
+                description="Minimum mapping quality for an alignment to be used.",
+            )
+            min_bq = IntegerField(
+                label="Minimum base quality",
+                required=False,
+                description="Minimum base quality for a base to be considered.",
+            )
+            excl_flags = StringField(
+                label="Filter flags",
+                required=False,
+                description="Filter flags: skip reads with mask bits set "
+                "[UNMAP,SECONDARY,QCFAIL,DUP]",
+            )
+            depth = IntegerField(
+                label="Maximum allowed coverage depth",
+                required=False,
+                description="If 0, depth is set to the maximum integer value "
+                "effectively removing any depth limit.",
```

Comment:
Depth limit is 1000000 by default. This parameter should have a matching default or mention of this in the description.